frontend/packages/ceph-storage-plugin/src/components/ocs-install/ocs-install.scss

```diff
@@ -29,3 +29,8 @@
 .ceph-install-select-mode {
   padding: var(--pf-global--spacer--sm) 0;
 }
+
+.ceph-ocs-co-required::after {
+  top: auto;
+  bottom: 0;
```

Comment:
Why `bottom: 0` ?

Reply:
since, the text expanded to 2 lines, hence `bottom:0` will work. Feel free to try it out.